File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
      height   = get_int("Height: ");
    }
    while(height<1|| height>8);

    for (int i=0;i<height;i++){
        for (int j=0;j<height-1-i;j++){
            printf(" ");
        }
        for (int h=0; h<=i;h++){
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
# TODO
import cs50


def main():
    input1 = 0
    while (input1<1 or input1 >8):
        input1 = cs50.get_int("Height:")
    recurs_pyramid(input1,input1)


def recurs_pyramid(n,i):
    if (i>0):
        recurs_pyramid(n,i-1)
        print(" "*(n-i),end="")
        print("#"*(i),end="")
        print("  ",end="")
        print("#"*(i))


if __name__ == "__main__":
    main()



